docs/src/pages/_app.page.tsx

```diff
@@ -1,5 +1,4 @@
 import Head from 'next/head';
-import '@aws-amplify/ui/dist/style.css';
```

Comment:
Ah, yes this should've been `@aws-amplify/ui/styles.css`:

https://github.com/aws-amplify/amplify-ui/blob/e9be592f172d6ac2613dd76123d5c5de47357b85/packages/ui/package.json#L8

But since it's duplicated by `ui-react/styles.css`, no need for it here.